PushSharp.Tests/PushSharp.Tests.csproj

```diff
@@ -1,74 +1,29 @@
+    <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
     <OutputPath>bin\Debug</OutputPath>
     <DefineConstants>DEBUG;</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>full</DebugType>
-    <Optimize>true</Optimize>
     <OutputPath>bin\Release</OutputPath>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <ConsolePause>false</ConsolePause>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.7.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="TestServiceConnection.cs" />
-    <Compile Include="Servers\TestApnsServer.cs" />
-    <Compile Include="ApnsTests.cs" />
-    <Compile Include="Settings.cs" />
-    <Compile Include="GcmRealTests.cs" />
-    <Compile Include="BrokerTests.cs" />
-    <Compile Include="ApnsRealTest.cs" />
-    <Compile Include="GcmTests.cs" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
   <ItemGroup>
-    <ProjectReference Include="..\PushSharp.Core\PushSharp.Core.csproj">
-      <Project>{2B44A8DA-60BC-4577-A2D7-D9D53F164B2E}</Project>
-      <Name>PushSharp.Core</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\PushSharp.Apple\PushSharp.Apple.csproj">
-      <Project>{A9D99F80-FEEB-4E74-96C5-66F17103C773}</Project>
-      <Name>PushSharp.Apple</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\PushSharp.Google\PushSharp.Google.csproj">
-      <Project>{94F16497-471F-433F-A99E-C455FB2D7724}</Project>
-      <Name>PushSharp.Google</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\PushSharp.Core\PushSharp.Core.csproj" />
+    <ProjectReference Include="..\PushSharp.Apple\PushSharp.Apple.csproj" />
+    <ProjectReference Include="..\PushSharp.Google\PushSharp.Google.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
     <None Include="settings.json" />
   </ItemGroup>
-  <ItemGroup />
   <ItemGroup>
-    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+    <Compile Remove="ApnsHttp2RealTests.cs" />
+    <Compile Remove="GcmXmppTests.cs" />
```

Comment:
@aarani remove? why?

Reply:
Because they were not included in the original project but since new proj includes folders you need to remove it manually

Reply:
why were they not included? o_O

Reply:
if you include them the build fails?

Reply:
The gcm result can be included if you include the removed stuff from .Google project as well, the Apple one (the part in .Apple project) requires HttpTwo project and that is .NET framework project and since we plan to remove that anyway I suggest we don't bother

Reply:
ok can you just say in the commit message that those files with Remove= were not included in the original .NET4.x projects please, then I merge

Reply:
Done